ui/src/frontend/README.md

```diff
@@ -0,0 +1,24 @@
+# frontend
```

Comment:
I assume this is pre-generated? It does contain the `npm install` I mentioned earlier, could be merged into the main `README.md` in the root and removed or the main README should point to this one and the incorrect info should be deleted there.